src/solver/model_getters.jl

```diff
@@ -221,6 +221,9 @@ function get_radiation_mode(parsed_args, ::Type{FT}) where {FT}
     @assert idealized_h2o in (true, false)
     idealized_clouds = parsed_args["idealized_clouds"]
     @assert idealized_clouds in (true, false)
+    prescribed_clouds = parsed_args["prescribed_clouds_in_radiation"]
+    @assert prescribed_clouds in (true, false)
+    @assert !(idealized_clouds && prescribed_clouds)
```

Comment:
Can you add an error message when this happens?

Can also add a message to warn when this is activated but the radiation model is not RRTGMP?